src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,352 @@
+        <td>Give users context about what the SVG image is conveying.</td>
+      </tr>
+      <tr>
+        <td>Intended as an image source</td>
+        <td>Communicate users what the image is conveying.</td>
+        <td>If the image is an icon, give users a description. For example, alt=" Lightbulb icon."</td>
+      </tr>
+      <tr>
+        <td>Acting as the SVG source</td>
+        <td>Must have a title.</td>
+        <td>Help users understand what the images is trying to communicate.</td>
+      </tr>
+      <tr>
+        <td>An inline SVG</td>
+        <td>Add the alt-text using the (title=" ") element within the SVG image.</td>
+        <td>Give users a brief description or advisory information of the graphic.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h2 class="sprk-b-TypeDisplayTwo sprk-u-mbl sprk-u-Measure">Writing Effective Alt Text Copy</h2>
+
+The author determines what the alt-text should be depending on the usage, context, and content of the image.
+
+> As a UX Writer, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbl sprk-u-Measure">Guidelines</h3>
+
+- <strong>Determine what information or what the function of the image is.</strong> Copy should describe an image’s purpose or what information it communicates. 
+- <strong>Choose the appropriate alternative text for the image type. </strong> If the image has no informative value, treat them as decorative images. 
+- <strong>Describe the image as specifically as possible. </strong> Include the most valuable information at the beginning and provide meaningful descriptions. 
+- <strong>Keep it as short as possible. </strong> Be concise and avoid keyword stuffing.
+- <strong>Use punctuation.</strong>
+<ul class="sprk-b-List sprk-b-List--indented"
+    data-id="unordered-list-2">
+    <li>Use sentence case.</li>
+    <li>Add space characters between the image and adjacent text to avoid having words running together when a screen reader reads them.</li>
+    <li>Include a period at the end of the alt-text – even if it's not a complete sentence. The period will provide a slight pause when read by the screen reader.</li>
+    <li>If using a null (empty) alternative text (alt="") to hide decorative images, make sure that there is no space character between the quotes.</li>
+</ul>
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">What Not To Do</h3>
+
+- Don’t use full sentences unless needed for clarity. 
+- Don’t include phrases like “image of” or “picture of.” 
+- Don’t include copyright or source information.
+- Don’t include supplementary information about the image. 
+- Don’t use the alt tag as a SEO opportunity. You can include keywords if they help describe the image, but don’t use the alt tag to stuff keywords into the screen.
```

Comment:
This is mentioned above. Do we need it in both spots?

Reply:
Removed the "What Not To Do" section so it's not repetitive